### DISCUSSION TITLE
Copilot Pro not activated

### DISCUSSION THREAD

**Discussion by njabulobc** [community/community#148156] [Copilot Conversations]
### Select Topic Area

Question

### Body

I was having challenges processing the payment for Github Copilot, until I got a message that it has been paid, the money deducted from my wallet, and 5 days later, not activated. I don't know whether this is an issue due to the new free copilot or its the old sync issue. In any case, Github support is NOT helpful, at all!

**Comment by Akash1134** [comment]
Hi @njabulobc ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
